jboss-image-streams.json

```diff
@@ -828,7 +840,8 @@
             "metadata": {
                 "name": "jboss-processserver64-openshift",
                 "annotations": {
-                    "openshift.io/display-name": "Red Hat JBoss BPM Suite 6.4 intelligent process server",
+                    "openshift.io/display-name": "Red Hat JBoss BPM Suite 6.3 intelligent process server",
```

Comment:
Inadvertent change?  It is v6.4.

Reply:
whoops, don't know how that happened. inadvertent